.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.7.1
+ruby-2.7.2
```

Comment:
curious - how did this affect you? did something break? was it just a clean up thing? should it be part of this PR?

Reply:
shouldn't necessarily be part of this PR, but I thought I might as well make it a part because when I run bundle, it tells me that ruby-2.7.1 isn't installed (if I installed it, it would tell me that my ruby version is 2.7.1, but my gemfile says 2.7.2)